src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * Class that detects if given post
+ * is displaying the Attendee List.
+ */
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets__Main;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+/**
+ * Class Attendee_List_Display
+ *
+ * @since   4.12.0
+ *
+ * @package Tribe\Tickets\Events
+ */
+class Attendee_List_Display {
```

Comment:
We should have the same class name as in ET+, `Attendees_List`. So that we can eventually make ET+ extend this one.